.forms.yml

```diff
@@ -0,0 +1,342 @@
+        default: "master"
+      - name: "Code git key"
+        description: "SSH key pair to fetch terraform source code Git repository."
+        key: git_private_key
+        type: string
+        widget: cy_cred
+        default: "((ssh_config.ssh_key))"
+      - name: "Code terraform path"
+        description: "Path of Terraform files in the git repository."
+        key: git_terraform_path
+        widget: simple_text
+        type: string
+        default: "./"
+
+
+s3-compatible:
+  pipeline:
+    Terraform:
+      - name: "AWS access key"
+        description: "Amazon AWS access key for Terraform."
+        key: aws_access_key
+        widget: cy_cred
+        type: string
+        default: "((aws.access_key))"
+      - name: "AWS secret key"
+        description: "Amazon AWS secret key for Terraform."
+        key: aws_secret_key
+        widget: cy_cred
+        type: string
+        default: "((aws.secret_key))"
+      - name: "Amazon AWS region"
+        description: "Amazon AWS region to use for Terraform."
+        key: aws_default_region
+        widget: auto_complete
+        type: string
+        values: ["eu-north-1","ap-south-1","eu-west-3","eu-west-2","eu-west-1","ap-northeast-2","ap-northeast-1","sa-east-1","ca-central-1","ap-southeast-1","ap-southeast-2","eu-central-1","us-east-1","us-east-2","us-west-1","us-west-2"]
+        default: eu-west-1
+      - name: "Terraform storage bucket"
+        description: "AWS S3 bucket name to store terraform remote state file."
+        key: terraform_storage_bucket_name
+        widget: simple_text
+        type: string
+        default: ($ organization_canonical $)-terraform-remote-state
+      - name: "Private module SSH key"
+        description: "SSH key to use to clone private terraform modules from Git."
+        key: terraform_modules_private_key
+        widget: cy_cred
+        type: string
+        default: ""
+      - name: "Terraform version"
+        description: "terraform version used to execute your code."
+        key: terraform_version
+        widget: simple_text
+        type: string
+        default: "latest"
+      - name: "S3 endpoint"
+        description: "A custom endpoint for the S3 API."
+        key: terraform_s3_endpoint
+        widget: simple_text
+        type: string
+        default: "http://10.10.10.10:9000"
+      - name: "S3 skip credentials validation"
+        description: "Skip the credentials validation via the STS API."
+        key: terraform_s3_skip_credentials_validation
+        widget: dropdown
+        type: boolean
+        default: true
+        values:
+            - true
+            - false
+      - name: "S3 force path style"
+        description: "Always use path-style S3 URLs (https://<HOST>/<BUCKET> instead of https://<BUCKET>.<HOST>)."
+        key: terraform_s3_force_path_style
+        widget: dropdown
+        type: boolean
+        default: true
+        values:
+            - true
+            - false
```

Comment:
```suggestion
        description: "Always use path-style S3 URLs (https://<HOST>/<BUCKET> instead of https://<BUCKET>.<HOST>)."
        key: terraform_s3_force_path_style
        widget: switch
        type: boolean
        default: true
```